docs/source/starthere/intro.rst

```diff
@@ -19,181 +19,126 @@ Conversational AI architectures are typically large and require a lot of data an
 
-From source
-~~~~~~~~~~~
-Use this installation mode if you are contributing to NeMo.
+Introductory videos
+-------------------
 
-.. code-block:: bash
-
-    apt-get update && apt-get install -y libsndfile1 ffmpeg
-    git clone https://github.com/NVIDIA/NeMo
-    cd NeMo
-    ./reinstall.sh
-
-Docker containers
-~~~~~~~~~~~~~~~~~
-To build a nemo container with Dockerfile from a branch,  please run 
-
-.. code-block:: bash
+See the two introductory videos below for a high level overview of NeMo.
 
-    DOCKER_BUILDKIT=1 docker build -f Dockerfile -t nemo:latest.
+**Developing State-Of-The-Art Conversational AI Models in Three Lines of Code**
 
+.. raw:: html
 
-If you chose to work with the ``main`` branch, we recommend using `NVIDIA's PyTorch container version 21.05-py3 <https://ngc.nvidia.com/containers/nvidia:pytorch/tags>`_, then install from GitHub.
+    <div style="position: relative; padding-bottom: 3%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/wBgpMf_KQVw" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    </div>
 
-.. code-block:: bash
+**NVIDIA NeMo: Toolkit for Conversational AI at PyData Yerevan 2022**
 
-    docker run --gpus all -it --rm -v <nemo_github_folder>:/NeMo --shm-size=8g \
-    -p 8888:8888 -p 6006:6006 --ulimit memlock=-1 --ulimit \
-    stack=67108864 --device=/dev/snd nvcr.io/nvidia/pytorch:21.05-py3
+.. raw:: html
 
-.. _mac-installation:
+    <div style="position: relative; padding-bottom: 3%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/J-P6Sczmas8?mute=0&start=14&autoplay=0" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    </div>
 
-Mac computers with Apple silicon
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-To install NeMo on Mac with Apple M-Series GPU:
+.. _installation:
 
-- install `Homebrew <https://brew.sh>`_ package manager
+Installation
+------------
 
-- create a new Conda environment
+The simplest way to install NeMo is via pip, see info below. 
 
-- install PyTorch 2.0 or higher
+.. note:: Full NeMo installation instructions (with more ways to install NeMo, and how to handle optional dependencies) can be found in the `GitHub README <https://github.com/NVIDIA/NeMo#installation>`_.
 
-- run the following code:
+Conda
+~~~~~
 
-.. code-block:: shell
+We recommend installing NeMo in a fresh Conda environment.
 
-    # install mecab using Homebrew, required for sacrebleu for NLP collection
-    brew install mecab
+.. code-block:: bash
 
-    # install pynini using Conda, required for text normalization
-    conda install -c conda-forge pynini
+    conda create --name nemo python==3.10.12
```

Comment:
Hmm do we need to be so specific? How about 3.10?

Reply:
I just copied what's in the README right now: https://github.com/NVIDIA/NeMo#conda